vmdb/app/models/hosts_storages.rb

```diff
@@ -1,3 +1,3 @@
 class HostsStorages < ActiveRecord::Base
-  table_name = "hosts_storages"
+  self.table_name = "hosts_storages"
```

Comment:
I'd keep it, just to make clear what the inflector is doing... seeing such a weird class name, personally, I'd have to check the DB to confirm what table name it was ending up with. (Though I don't foresee the inflector's behaviour changing any time soon, even for such edge cases.)